rtsf-at-checkout-event-reconciler/main.go

```diff
@@ -59,9 +60,7 @@ func (app *EventReconcilerAppService) CreateAndRunAppService(serviceKey string)
 	eventsProcessor.ResetEventsOccurrence()
 	eventsProcessor.InitWebSocketConnection(app.service, app.lc)
 
-	deviceNamesList := strings.TrimSpace(app.serviceConfig.Reconciler.DeviceNames)
-
-	deviceNames := strings.Split(deviceNamesList, ",")
+	deviceNames := util.DeleteEmptyAndTrim(strings.FieldsFunc(app.serviceConfig.Reconciler.DeviceNames, util.SplitComma))
```

Comment:
Fix code that was parsing the device names, which left a blank at the start of each name so the filter wasn't accepting any of the data from the devices.